src/app/vitessce-grid-utils.js

```diff
@@ -104,7 +104,14 @@ export function createLoaders(datasets, configDescription) {
       // Fall back to JsonLoader if a loader is not found for the file type.
       const matchingLoaderClass = fileTypeToLoader[file.fileType] || JsonLoader;
       // eslint-disable-next-line new-cap
-      datasetLoaders.loaders[file.type] = new matchingLoaderClass(file);
+      const loader = new matchingLoaderClass(file);
+      if (Array.isArray(file.type)) {
```

Comment:
Perhaps this is _only_ needed when the `fileType` does not imply this because they are one-to-one